selfdrive/car/chrysler/carcontroller.py

```diff
@@ -45,10 +46,14 @@ def update(self, enabled, CS, actuators, pcm_cancel_cmd, hud_alert):
     can_sends = []
 
     #*** control msgs ***
+    
+    wheel_button_counter_change = CS.wheel_button_counter != self.wheel_button_counter_prev
+    if wheel_button_counter_change:
+      self.wheel_button_counter_prev = CS.wheel_button_counter
 
-    if pcm_cancel_cmd:
+    if pcm_cancel_cmd and (self.ccframe % 10 < 5) and wheel_button_counter_change:
```

Comment:
what does `self.ccframe % 10 < 5` do?

Reply:
sends cancel for 50msec and gives a 50msec break before resending for 50msec

Reply:
how often is the first message not accepted?

Reply:
don't have any data, but have had issue with continuous spamming.

Reply:
can you try removing the that condition? I'd also like some information on how often the first cancel isn't taken.

Reply:
alright.  u probably can gather that info from all the chrysler users. my vehicle is a my21 pacifica and it had more issues with than older pacificas.

Reply:
It shouldn't be needed as the cancel button is accepted on the down press and not up.  We should continue to send the press until OP disengages.

Reply:
thanks

Reply:
I haven't tested that.  But I know you can physically hold down buttons on the wheel without faults.

Reply:
ya, but the steering wheel is already sending a msg continuously and OP is trying to double send the same message when OP wants to send cancel, so the receiving module may fault out if it see discrepancy in the counter at some point. because it is see a counter value from steering button and OP. we may not see any issue if the cancel from OP got accepted quickly, which is what we are mostly seeing on our vehicles.